.changes/unreleased/ENHANCEMENTS-20250205-104144.yaml

```
@@ -0,0 +1,5 @@
+kind: ENHANCEMENTS
+body: Allow terraform init when only test files are present in directory
```

This is a nit, but I found this change message ambiguous as someone who is less familiar with `terraform test`, because there's both the "current directory" and the "testing directory" to think about.

This is the best alternative I could think of

```suggestion
body: Allow terraform init when tests are present in the testing directory but no configuration files are directly inside the current directory
```